e2e/support/helpers/e2e-action-helpers.js

```diff
@@ -1,5 +1,7 @@
 import { capitalize } from "inflection";
 
+import { nativeEditor } from "e2e/support/helpers/e2e-native-editor-helpers";
```

Comment:
Even from files in `e2e/support/helpers`, that would lead to import loops.

Other files in `e2e/support/helpers` do not do this either, eg. [`e2e/support/helpers/e2e-embedding-sdk-helpers.ts`](https://github.com/metabase/metabase/blob/master/e2e/support/helpers/e2e-embedding-sdk-helpers.ts#L1-L3).

Reply:
> Even from files in `e2e/support/helpers`? I'm asking since that would lead to import loops.

Right, files inside `e2e/support/helpers` should use relative imports for other helpers